.circleci/config.yml

```diff
@@ -124,7 +130,10 @@ jobs:
           package: nitro-protocol
 
   integration-test:
-    <<: *defaults
+    resource_class: medium
+    working_directory: /home/circleci/project
+    docker:
+      - image: circleci/node:10.16.3
```

Comment:
What's the reason a different image is used for integration tests vs other jobs?

Reply:
The `counterfactual:statechannels` image is built ontop of `node:10.16-slim`, and to be honest I did not test it, but I suspect `puppeteer` won't run on there, and I did not want to deal with that at the moment. I know `circleci/node:10.16.3` seems to be fine with `puppeteer`, so just took the easy path.

Reply:
Makes sense!